lib/matplotlib/__init__.py

```diff
@@ -207,14 +207,35 @@ def _check_versions():
     sys.argv = ['modpython']
 
 
-_verbose_msg = """\
-matplotlib.verbose is deprecated;
-Command line argument --verbose-LEVEL is deprecated.
-This functionality is now provided by the standard
-python logging library.  To get more (or less) logging output:
-    import logging
-    logger = logging.getLogger('matplotlib')
-    logger.set_level(logging.INFO)"""
+def set_loglevel(level='info'):
```

Comment:
Do we need a default? Just from `set_loglevel()` I couldn't guess what level is now in use. Having to pass the level makes it explicit.

Reply:
I disagree with this.  The only reason it is ambiguous is you suggested we change the name from verbose :wink:   The naive user posting on stack overflow is not going to know the names of the logging levels so having an interface where they don’t have to pass an argument is desirable.